### DISCUSSION TITLE
suggestions for an actual working good speach tool without internetconnections?

### DISCUSSION THREAD

**Discussion by kalle07** [oobabooga/text-generation-webui#6507] [Q&A]
suggestions for an actual working good speach tool without internet connections?

alltalk was the best so fare ... at the moment under cunstructions

all other where bad - slow or had short internet-connection, you can proof if your disable your lan

**Comment by finefin** [comment]
AllTalk v2 is out since end of november and it works great!

**Comment by kalle07** [comment]
yes, a bit hard to install but it runs  ;)
